components/wpt-results.html

```diff
@@ -284,6 +287,20 @@
       _dateFormat(dateStr) {
         return String(new Date(dateStr)).match(/^\w+ (\w+ \w+ \w+)/)[1]
       }
+
+      _testFileHSL(result) {
+        // Need saturation between 65-100, reversed (range 35)
+        // hsl(129, [65-100]%, 65%)
+        const passRate = result.passing / result.total
+        if (passRate === 1) {
+          // Green
+          return 'background-color: hsl(129, 85%, 65%)'
+        } else {
+          const luminance = 65 + passRate * 20
+          // Some shade of red
+          return `background-color: hsl(0, 85%, ${luminance}%)`
```

Comment:
Hmm, so hue = 0 is red, and saturation is the same either way, and we're changing from dark red to light red by using a luminance between 65% and 85%?